stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -82,6 +82,7 @@
  * the {@code stetho-sample} for more information.
  */
 public class Stetho {
+  public static String password = "";
```

Comment:
This is ugly but effective, no need to pass it all the way down to the correct class